### PR TITLE
PR: Set Python as default file extension for the "Save As" dialog

### PR DESCRIPTION
**Pull request by malliwi88**
Take a look to this @ccordoba12 and @goanpeca.

This PR can solve your problems @rlaverde with save_as feature.

any suggestions  and comments are welcome. :+1:

### CODE REVIEW

File: spyder/config/utils.py [hunk 1]
```diff
@@ -116,8 +116,10 @@ def get_filter(filetypes, ext):
 def get_edit_filetypes():
     """Get all file types supported by the Editor"""
     pygments_exts = _get_pygments_extensions()
-    other_exts = ['.ipynb', '.md']
-    all_exts = tuple(pygments_exts + other_exts)
+    favorite_exts = ['.py', '.R', '.jl', '.pyw', '.pyx', '.C', '.CPP']
+    other_exts = ['.ipynb', '.md'] + [ext for ext in pygments_exts if ext not in favorite_exts]
+    all_exts = tuple(favorite_exts + other_exts)
+    print(all_exts)
```
Comment (ccordoba12) [inline spyder/config/utils.py]: Please remove this `print`

File: spyder/config/utils.py [hunk 2]
```diff
@@ -116,8 +116,9 @@ def get_filter(filetypes, ext):
 def get_edit_filetypes():
     """Get all file types supported by the Editor"""
     pygments_exts = _get_pygments_extensions()
-    other_exts = ['.ipynb', '.md']
-    all_exts = tuple(pygments_exts + other_exts)
+    favorite_exts = ['.py', '.R', '.jl', '.pyw', '.pyx', '.C', '.CPP']
+    other_exts = ['.ipynb', '.md'] + [ext for ext in pygments_exts if ext not in favorite_exts]
```
Comment (ccordoba12) [inline spyder/config/utils.py]: Please move `.ipynb` and `.md` to `favorite_exts` (after `.jl`)

Comment (malliwi88) [inline spyder/config/utils.py]: Ok, great!!!

Comment (malliwi88) [inline spyder/config/utils.py]: at the second and third options? or It not relevant now?

**Comment by ccordoba12** [comment]
I think this is ready. Let's merge it and see how it goes :-)

Thanks @malliwi88!

**Pull request merged**